test/reporters/xunit.spec.js

```diff
@@ -194,15 +194,13 @@ describe('XUnit reporter', function() {
       it('should call write with line', function() {
         stdoutWrite = process;
         process = false; // eslint-disable-line no-native-reassign, no-global-assign
```

Comment:
Mentioned elsewhere, but this test should fail after `process` reassignment...

Reply:
Is there a change needed to this test to make that happen? Dont follow why `should fail after process reassignment` would be true.

Reply:
@plroebuck any chance you could help me with this please.

Reply:
https://github.com/mochajs/mocha/pull/2698#issuecomment-467413307

Reply:
```
node bin/mocha "test/reporters/xunit.spec.js"
```

Nope, locally passes for me on master and this branch.

Reply:
Is your concern running function on a falsy value? Seems as its re-assigned before the test finishes its back for `EVENT_RUN_END`.